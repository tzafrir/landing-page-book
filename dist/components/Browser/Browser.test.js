'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _enzymeAdapterReact = require('enzyme-adapter-react-16');

var _enzymeAdapterReact2 = _interopRequireDefault(_enzymeAdapterReact);

var _Browser = require('./Browser');

var _Browser2 = _interopRequireDefault(_Browser);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

(0, _enzyme.configure)({ adapter: new _enzymeAdapterReact2.default() });

describe('Browser', function () {
  var props = void 0;
  var mounted = void 0;
  var browser = function browser() {
    if (!mounted) {
      mounted = (0, _enzyme.shallow)(_react2.default.createElement(_Browser2.default, props));
    }
    return mounted;
  };
  describe('Browser', function () {
    beforeEach(function () {
      props = {};
      mounted = undefined;
    });

    beforeEach(function () {
      props = {
        vendor: 'chrome'
      };
      mounted = undefined;
    });

    it('should render', function () {
      expect((0, _enzyme.shallow)(_react2.default.createElement(_Browser2.default, props))).toMatchSnapshot();
    });

    it('always renders a div', function () {
      var divs = browser().find('div');

      expect(divs.length).toBeGreaterThan(0);
    });
  });
});