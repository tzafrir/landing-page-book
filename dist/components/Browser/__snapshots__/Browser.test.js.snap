// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Browser Browser should render 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Browser
    imgFilter={null}
    showBrowserVendor={true}
    showBrowserVersion={true}
    vendor="chrome"
    version=""
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Image
          alt="Browser Chrome icon"
          border={false}
          className="Browser__Icon"
          imgFilter={null}
          rounded={false}
          shadow={false}
          src="data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMjUuNzkgMTI2LjIyIj48ZGVmcz48c3R5bGU+LmNscy0xe2ZpbGw6I2VmM2YzNjt9LmNscy0ye2ZpbGw6I2ZjZDkwMDt9LmNscy0ze2ZpbGw6IzYxYmM1Yjt9LmNscy00e2ZpbGw6IzVhYjA1NTt9LmNscy01e2ZpbGw6I2ZmZjt9LmNscy02e2ZpbGw6IzRkOGNmNDt9LmNscy03e2ZpbGw6I2VhY2EwNTt9LmNscy04e2ZpbGw6I2RmM2EzMjt9PC9zdHlsZT48L2RlZnM+PHRpdGxlPmNocm9tZTwvdGl0bGU+PHBhdGggY2xhc3M9ImNscy0xIiBkPSJNNjIuNzgsMHMzNy0xLjY3LDU2LjY3LDM1LjY0SDU5LjU4UzQ4LjI4LDM1LjI1LDM4LjYzLDQ5Yy0yLjc4LDUuNzUtNS43NiwxMS43LTIuNDEsMjMuMzZDMzEuNCw2NC4xNSwxMC42NCwyOCwxMC42NCwyOFMyNS4yOCwxLjQ5LDYyLjc4LDBaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtMiIgZD0iTTExNy41Myw5NC42cy0xNy4xMSwzMy01OS4xOCwzMS4zMkw4OC4yOSw3NC4wN3M2LTkuNi0xLjA3LTI0LjgyQzgzLjYzLDQ0LDgwLDM4LjQ0LDY4LjIyLDM1LjQ5YzkuNDgtLjA5LDUxLjE2LDAsNTEuMTYsMFMxMzUsNjEuNDMsMTE3LjUzLDk0LjZaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtMyIgZD0iTTguMTksOTQuODZzLTIwLTMxLjI4LDIuNDgtNjYuOTFDMTUuODUsMzcsNDAuNiw3OS44LDQwLjYsNzkuOHM1LjM0LDEwLDIyLDExLjQ4YzYuMzYtLjQ3LDEzLS44NywyMS40NC05LjU5Qzc5LjM3LDg5Ljk0LDU4LjQ2LDEyNiw1OC40NiwxMjZzLTMwLjMuNTUtNTAuMy0zMS4xM1oiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuMDEgLTAuMDEpIi8+PHBhdGggY2xhc3M9ImNscy00IiBkPSJNNTguMzMsMTI2LjIzbDguNDUtMzUuMTRhMjguMiwyOC4yLDAsMCwwLDE3LTkuMjRDNzksOTAuMzIsNTguMzQsMTI2LjIzLDU4LjM0LDEyNi4yM1oiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuMDEgLTAuMDEpIi8+PHBhdGggY2xhc3M9ImNscy01IiBkPSJNMzQuNTIsNjMuNTFBMjguMDYsMjguMDYsMCwxLDEsNjIuNTgsOTEuNTcsMjguMDYsMjguMDYsMCwwLDEsMzQuNTIsNjMuNTFaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtNiIgZD0iTTM5LjIyLDYzLjUxQTIzLjM2LDIzLjM2LDAsMSwxLDYyLjU4LDg2Ljg3aDBBMjMuMzYsMjMuMzYsMCwwLDEsMzkuMjIsNjMuNTFaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtNyIgZD0iTTExOS4zMywzNS41MSw4NC42NSw0NS42NkEyOC4xMywyOC4xMywwLDAsMCw2OC4xNywzNS41MUM3Ny45MywzNS40NiwxMTkuMzMsMzUuNTEsMTE5LjMzLDM1LjUxWiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMSAtMC4wMSkiLz48cGF0aCBjbGFzcz0iY2xzLTgiIGQ9Ik0zNS41NCw3MWMtNC44Ny04LjQ0LTI0LjktNDMtMjQuOS00M0wzNi4zMiw1My40MUEyNS4zMSwyNS4zMSwwLDAsMCwzNC42OCw2Ni42WiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMSAtMC4wMSkiLz48L3N2Zz4="
          tooltip=""
          width={undefined}
        />,
        <div
          className="Browser__Text"
        >
           
          Chrome
        </div>,
        <div
          className="Browser__Version"
        >
          
        </div>,
      ],
      "className": "d-inline Browser",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "alt": "Browser Chrome icon",
          "border": false,
          "className": "Browser__Icon",
          "imgFilter": null,
          "rounded": false,
          "shadow": false,
          "src": "data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMjUuNzkgMTI2LjIyIj48ZGVmcz48c3R5bGU+LmNscy0xe2ZpbGw6I2VmM2YzNjt9LmNscy0ye2ZpbGw6I2ZjZDkwMDt9LmNscy0ze2ZpbGw6IzYxYmM1Yjt9LmNscy00e2ZpbGw6IzVhYjA1NTt9LmNscy01e2ZpbGw6I2ZmZjt9LmNscy02e2ZpbGw6IzRkOGNmNDt9LmNscy03e2ZpbGw6I2VhY2EwNTt9LmNscy04e2ZpbGw6I2RmM2EzMjt9PC9zdHlsZT48L2RlZnM+PHRpdGxlPmNocm9tZTwvdGl0bGU+PHBhdGggY2xhc3M9ImNscy0xIiBkPSJNNjIuNzgsMHMzNy0xLjY3LDU2LjY3LDM1LjY0SDU5LjU4UzQ4LjI4LDM1LjI1LDM4LjYzLDQ5Yy0yLjc4LDUuNzUtNS43NiwxMS43LTIuNDEsMjMuMzZDMzEuNCw2NC4xNSwxMC42NCwyOCwxMC42NCwyOFMyNS4yOCwxLjQ5LDYyLjc4LDBaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtMiIgZD0iTTExNy41Myw5NC42cy0xNy4xMSwzMy01OS4xOCwzMS4zMkw4OC4yOSw3NC4wN3M2LTkuNi0xLjA3LTI0LjgyQzgzLjYzLDQ0LDgwLDM4LjQ0LDY4LjIyLDM1LjQ5YzkuNDgtLjA5LDUxLjE2LDAsNTEuMTYsMFMxMzUsNjEuNDMsMTE3LjUzLDk0LjZaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtMyIgZD0iTTguMTksOTQuODZzLTIwLTMxLjI4LDIuNDgtNjYuOTFDMTUuODUsMzcsNDAuNiw3OS44LDQwLjYsNzkuOHM1LjM0LDEwLDIyLDExLjQ4YzYuMzYtLjQ3LDEzLS44NywyMS40NC05LjU5Qzc5LjM3LDg5Ljk0LDU4LjQ2LDEyNiw1OC40NiwxMjZzLTMwLjMuNTUtNTAuMy0zMS4xM1oiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuMDEgLTAuMDEpIi8+PHBhdGggY2xhc3M9ImNscy00IiBkPSJNNTguMzMsMTI2LjIzbDguNDUtMzUuMTRhMjguMiwyOC4yLDAsMCwwLDE3LTkuMjRDNzksOTAuMzIsNTguMzQsMTI2LjIzLDU4LjM0LDEyNi4yM1oiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuMDEgLTAuMDEpIi8+PHBhdGggY2xhc3M9ImNscy01IiBkPSJNMzQuNTIsNjMuNTFBMjguMDYsMjguMDYsMCwxLDEsNjIuNTgsOTEuNTcsMjguMDYsMjguMDYsMCwwLDEsMzQuNTIsNjMuNTFaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtNiIgZD0iTTM5LjIyLDYzLjUxQTIzLjM2LDIzLjM2LDAsMSwxLDYyLjU4LDg2Ljg3aDBBMjMuMzYsMjMuMzYsMCwwLDEsMzkuMjIsNjMuNTFaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtNyIgZD0iTTExOS4zMywzNS41MSw4NC42NSw0NS42NkEyOC4xMywyOC4xMywwLDAsMCw2OC4xNywzNS41MUM3Ny45MywzNS40NiwxMTkuMzMsMzUuNTEsMTE5LjMzLDM1LjUxWiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMSAtMC4wMSkiLz48cGF0aCBjbGFzcz0iY2xzLTgiIGQ9Ik0zNS41NCw3MWMtNC44Ny04LjQ0LTI0LjktNDMtMjQuOS00M0wzNi4zMiw1My40MUEyNS4zMSwyNS4zMSwwLDAsMCwzNC42OCw2Ni42WiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMSAtMC4wMSkiLz48L3N2Zz4=",
          "tooltip": "",
          "width": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            " ",
            "Chrome",
          ],
          "className": "Browser__Text",
        },
        "ref": null,
        "rendered": Array [
          " ",
          "Chrome",
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "",
          "className": "Browser__Version",
        },
        "ref": null,
        "rendered": "",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Image
            alt="Browser Chrome icon"
            border={false}
            className="Browser__Icon"
            imgFilter={null}
            rounded={false}
            shadow={false}
            src="data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMjUuNzkgMTI2LjIyIj48ZGVmcz48c3R5bGU+LmNscy0xe2ZpbGw6I2VmM2YzNjt9LmNscy0ye2ZpbGw6I2ZjZDkwMDt9LmNscy0ze2ZpbGw6IzYxYmM1Yjt9LmNscy00e2ZpbGw6IzVhYjA1NTt9LmNscy01e2ZpbGw6I2ZmZjt9LmNscy02e2ZpbGw6IzRkOGNmNDt9LmNscy03e2ZpbGw6I2VhY2EwNTt9LmNscy04e2ZpbGw6I2RmM2EzMjt9PC9zdHlsZT48L2RlZnM+PHRpdGxlPmNocm9tZTwvdGl0bGU+PHBhdGggY2xhc3M9ImNscy0xIiBkPSJNNjIuNzgsMHMzNy0xLjY3LDU2LjY3LDM1LjY0SDU5LjU4UzQ4LjI4LDM1LjI1LDM4LjYzLDQ5Yy0yLjc4LDUuNzUtNS43NiwxMS43LTIuNDEsMjMuMzZDMzEuNCw2NC4xNSwxMC42NCwyOCwxMC42NCwyOFMyNS4yOCwxLjQ5LDYyLjc4LDBaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtMiIgZD0iTTExNy41Myw5NC42cy0xNy4xMSwzMy01OS4xOCwzMS4zMkw4OC4yOSw3NC4wN3M2LTkuNi0xLjA3LTI0LjgyQzgzLjYzLDQ0LDgwLDM4LjQ0LDY4LjIyLDM1LjQ5YzkuNDgtLjA5LDUxLjE2LDAsNTEuMTYsMFMxMzUsNjEuNDMsMTE3LjUzLDk0LjZaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtMyIgZD0iTTguMTksOTQuODZzLTIwLTMxLjI4LDIuNDgtNjYuOTFDMTUuODUsMzcsNDAuNiw3OS44LDQwLjYsNzkuOHM1LjM0LDEwLDIyLDExLjQ4YzYuMzYtLjQ3LDEzLS44NywyMS40NC05LjU5Qzc5LjM3LDg5Ljk0LDU4LjQ2LDEyNiw1OC40NiwxMjZzLTMwLjMuNTUtNTAuMy0zMS4xM1oiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuMDEgLTAuMDEpIi8+PHBhdGggY2xhc3M9ImNscy00IiBkPSJNNTguMzMsMTI2LjIzbDguNDUtMzUuMTRhMjguMiwyOC4yLDAsMCwwLDE3LTkuMjRDNzksOTAuMzIsNTguMzQsMTI2LjIzLDU4LjM0LDEyNi4yM1oiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuMDEgLTAuMDEpIi8+PHBhdGggY2xhc3M9ImNscy01IiBkPSJNMzQuNTIsNjMuNTFBMjguMDYsMjguMDYsMCwxLDEsNjIuNTgsOTEuNTcsMjguMDYsMjguMDYsMCwwLDEsMzQuNTIsNjMuNTFaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtNiIgZD0iTTM5LjIyLDYzLjUxQTIzLjM2LDIzLjM2LDAsMSwxLDYyLjU4LDg2Ljg3aDBBMjMuMzYsMjMuMzYsMCwwLDEsMzkuMjIsNjMuNTFaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtNyIgZD0iTTExOS4zMywzNS41MSw4NC42NSw0NS42NkEyOC4xMywyOC4xMywwLDAsMCw2OC4xNywzNS41MUM3Ny45MywzNS40NiwxMTkuMzMsMzUuNTEsMTE5LjMzLDM1LjUxWiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMSAtMC4wMSkiLz48cGF0aCBjbGFzcz0iY2xzLTgiIGQ9Ik0zNS41NCw3MWMtNC44Ny04LjQ0LTI0LjktNDMtMjQuOS00M0wzNi4zMiw1My40MUEyNS4zMSwyNS4zMSwwLDAsMCwzNC42OCw2Ni42WiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMSAtMC4wMSkiLz48L3N2Zz4="
            tooltip=""
            width={undefined}
          />,
          <div
            className="Browser__Text"
          >
             
            Chrome
          </div>,
          <div
            className="Browser__Version"
          >
            
          </div>,
        ],
        "className": "d-inline Browser",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "alt": "Browser Chrome icon",
            "border": false,
            "className": "Browser__Icon",
            "imgFilter": null,
            "rounded": false,
            "shadow": false,
            "src": "data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMjUuNzkgMTI2LjIyIj48ZGVmcz48c3R5bGU+LmNscy0xe2ZpbGw6I2VmM2YzNjt9LmNscy0ye2ZpbGw6I2ZjZDkwMDt9LmNscy0ze2ZpbGw6IzYxYmM1Yjt9LmNscy00e2ZpbGw6IzVhYjA1NTt9LmNscy01e2ZpbGw6I2ZmZjt9LmNscy02e2ZpbGw6IzRkOGNmNDt9LmNscy03e2ZpbGw6I2VhY2EwNTt9LmNscy04e2ZpbGw6I2RmM2EzMjt9PC9zdHlsZT48L2RlZnM+PHRpdGxlPmNocm9tZTwvdGl0bGU+PHBhdGggY2xhc3M9ImNscy0xIiBkPSJNNjIuNzgsMHMzNy0xLjY3LDU2LjY3LDM1LjY0SDU5LjU4UzQ4LjI4LDM1LjI1LDM4LjYzLDQ5Yy0yLjc4LDUuNzUtNS43NiwxMS43LTIuNDEsMjMuMzZDMzEuNCw2NC4xNSwxMC42NCwyOCwxMC42NCwyOFMyNS4yOCwxLjQ5LDYyLjc4LDBaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtMiIgZD0iTTExNy41Myw5NC42cy0xNy4xMSwzMy01OS4xOCwzMS4zMkw4OC4yOSw3NC4wN3M2LTkuNi0xLjA3LTI0LjgyQzgzLjYzLDQ0LDgwLDM4LjQ0LDY4LjIyLDM1LjQ5YzkuNDgtLjA5LDUxLjE2LDAsNTEuMTYsMFMxMzUsNjEuNDMsMTE3LjUzLDk0LjZaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtMyIgZD0iTTguMTksOTQuODZzLTIwLTMxLjI4LDIuNDgtNjYuOTFDMTUuODUsMzcsNDAuNiw3OS44LDQwLjYsNzkuOHM1LjM0LDEwLDIyLDExLjQ4YzYuMzYtLjQ3LDEzLS44NywyMS40NC05LjU5Qzc5LjM3LDg5Ljk0LDU4LjQ2LDEyNiw1OC40NiwxMjZzLTMwLjMuNTUtNTAuMy0zMS4xM1oiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuMDEgLTAuMDEpIi8+PHBhdGggY2xhc3M9ImNscy00IiBkPSJNNTguMzMsMTI2LjIzbDguNDUtMzUuMTRhMjguMiwyOC4yLDAsMCwwLDE3LTkuMjRDNzksOTAuMzIsNTguMzQsMTI2LjIzLDU4LjM0LDEyNi4yM1oiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuMDEgLTAuMDEpIi8+PHBhdGggY2xhc3M9ImNscy01IiBkPSJNMzQuNTIsNjMuNTFBMjguMDYsMjguMDYsMCwxLDEsNjIuNTgsOTEuNTcsMjguMDYsMjguMDYsMCwwLDEsMzQuNTIsNjMuNTFaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtNiIgZD0iTTM5LjIyLDYzLjUxQTIzLjM2LDIzLjM2LDAsMSwxLDYyLjU4LDg2Ljg3aDBBMjMuMzYsMjMuMzYsMCwwLDEsMzkuMjIsNjMuNTFaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjAxIC0wLjAxKSIvPjxwYXRoIGNsYXNzPSJjbHMtNyIgZD0iTTExOS4zMywzNS41MSw4NC42NSw0NS42NkEyOC4xMywyOC4xMywwLDAsMCw2OC4xNywzNS41MUM3Ny45MywzNS40NiwxMTkuMzMsMzUuNTEsMTE5LjMzLDM1LjUxWiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMSAtMC4wMSkiLz48cGF0aCBjbGFzcz0iY2xzLTgiIGQ9Ik0zNS41NCw3MWMtNC44Ny04LjQ0LTI0LjktNDMtMjQuOS00M0wzNi4zMiw1My40MUEyNS4zMSwyNS4zMSwwLDAsMCwzNC42OCw2Ni42WiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMSAtMC4wMSkiLz48L3N2Zz4=",
            "tooltip": "",
            "width": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              " ",
              "Chrome",
            ],
            "className": "Browser__Text",
          },
          "ref": null,
          "rendered": Array [
            " ",
            "Chrome",
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "",
            "className": "Browser__Version",
          },
          "ref": null,
          "rendered": "",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
